benchmarks/load_test/README.md

```diff
@@ -0,0 +1,22 @@
+### Load testing
+
+Create a Locust load testing instance by using the CLI tool `load_test.py`
+
+It *requires* locust to be installed, use `pip install locustio` (if dev-dependencies is not already installed).
+
+#### Usage
+
+- Start port-forwarding with *ambassador* on a cluster (with an existing gordo-project).
+
+    `kubectl port-forward svc/ambassador -n ambassador 8888:80`
+
+- Use the CLI tool `load_test.py`, specify the *project name* (required), *host* (optional) and *port* (optional).
```

Comment:
Should you add a short little example snippet here? something lazy people like myself can copy and paste :-)